diesel_codegen/src/lib.rs

```diff
@@ -1,4 +1,5 @@
 #![deny(warnings)]
+#![recursion_limit = "512"]
```

Comment:
The `quote!` macro makes use of (potentially deep) recursion, and my large invocations of it are running into the default limit. This number should be tuned to fit whatever this PR's eventual needs are; it may be that we wind up needing something smaller at first.